### DISCUSSION TITLE
Error: Not equal Type

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#932] [Q&A]
I have loaded the performance file Cessna Skyhawk G1000 Asobo from the performance site LNM.
Since 1 week or so the tab "Fuel Report" shows the error "User aircraft type "Skyhawk" in simulator is not equal to type "C172" used in performance file".
I tried to do an own performance flight, but the error was the same.
I have done a reset in LNM but no success.
I think that this error occurs after I have loaded the new WB SIM Mod for the classic C172, but I am flying with the C172_G1000.
I tried the same with the Diamond DA62 and there was no issue.
LNM Version 2.8.0 beta, MFS2020 latest version.
What can I do as next?
Thanks

**Comment by albar965** [comment]
The aircraft from the simulator reports the wrong [ICAO aircraft type designator](https://www.icao.int/publications/DOC8643/Pages/Search.aspx). 

You can change the aircraft performance file to use the wrong designator, simply ignore the message, contact the add-on developer to fix this or do this on your own by modifying the aircraft files.

See here for more information: https://www.littlenavmap.org/manuals/littlenavmap/release/2.6/en/AIRCRAFTPERFEDIT.html#tab-aircraft
